.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 sudo: false
 node_js:
-  - "6.10.3"
+  - "8.1.0"
```

Comment:
I concur. Most of it is the wait time for the Mac OS builds though – not much we can do about that (on Travis at least).